src/invidious/views/components/channel-information.ecr

```diff
@@ -0,0 +1,146 @@
+        <!-- TODO Refactor this! -->
+        <div class="pure-menu pure-menu-horizontal">
+          <ui class="pure-menu-list">
+              <% if !channel.auto_generated %>
+                  <% if content_type == 0 %>
+                      <li class="pure-menu-item pure-menu-selected">
+                        <a href="/channel/<%= channel.ucid %>" class="pure-menu-link">
+                          <b><%= translate(locale, "Videos") %></b>
+                        </a>
+                      </li>
+                  <% else %>
+                      <li class="pure-menu-item pure-menu">
+                        <a href="/channel/<%= channel.ucid %>" class="pure-menu-link">
+                          <%= translate(locale, "Videos") %>
+                        </a>
+                      </li>
+                  <% end %>
+              <% end %>
+
+              <% if content_type == 1 || channel.auto_generated %>
+                  <li class="pure-menu-item pure-menu-selected">
+                      <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/playlists">
+                        <b> <%= translate(locale, "Playlists") %> </b>
+                      </a>
+                  </li>
+              <% else %>
+                  <li class="pure-menu-item">
+                      <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/playlists">
+                        <%= translate(locale, "Playlists") %>
+                      </a>
+                  </li>
+              <% end %>
+
+              <% if channel.tabs.includes? "community" %>
+                      <% if content_type == 2 %>
+                          <li class="pure-menu-item pure-menu-selected">
+                              <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/community">
+                                <b> <%= translate(locale, "Community") %> </b>
+                              </a>
+                          </li>
+                      <% else %>
+                          <li class="pure-menu-item">
+                              <a class="pure-menu-link" href="/channel/<%= channel.ucid %>/community">
+                                <%= translate(locale, "Community") %>
+                              </a>
+                          </li>
+                      <% end %>
+              <% end %>
+
+              <% if !channel.auto_generated %>
+                  <li class="pure-menu-item navbar" style="margin: 0 0 0 0.5em">
+                      <div class="searchbar">
+                          <form class="pure-form" action="/search" method="get">
+                              <fieldset>
+                                  <button class="search-button"><i class="icon ion-md-search"></i></button>
+                                  <input type="search" name="q" placeholder="<%= translate(locale, "search") %>" value="<%= env.get?("search").try {|x| HTML.escape(x.as(String)) } %>">
```

Comment:
Again for acessibility: this input should have the attribute `type="text"` (`type="search"` is not a standard thing) and an `aria-label` to describe what it's for ("Channel search")